app/controllers/miq_ae_class_controller.rb

```diff
@@ -197,6 +198,13 @@ def get_node_info(node)
     x_history_add_item(:id => x_node, :text => @right_cell_text)
   end
 
+  def add_domain_version_message
+    version = @record.version
+    available_version = @record.available_version
+    return if version.nil? || available_version.nil?
+    @version_message = _("Currently running %s version, you have %s version available.") % [version, available_version] if version != available_version
```

Comment:
@gmcculloug this message in only displayed when user has selected MIQ or Red Hat domain in the tree, message is not displayed on any other node so i am not sure if showing a Domain name is needed.